src/nodes/corenodes/filter/edge_detection_node/edge_detection_node.py

```diff
@@ -0,0 +1,64 @@
+# ----------------------------------------------------------------------------
+# Gimel Studio Copyright 2019-2023 by the Gimel Studio project contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------
+
+from gimelstudio import api
+
+
+class EdgeDetectionNode(api.Node):
+    def __init__(self, nodegraph, id):
+        api.Node.__init__(self, nodegraph, id)
+
+    @property
+    def NodeMeta(self):
+        meta_info = {
+            "label": "Edge Detection",
+            "author": "Gimel Studio",
+            "version": (0, 5, 0),
```

Comment:
Bump version to `0.6.0`

```suggestion
            "version": (0, 6, 0),
```